specs/clear-site-data/index.html

```diff
@@ -1194,7 +1194,7 @@ <h4 class="heading settled" data-level="3.4.3" id="clear-cache"><span class="sec
   Chrome clears out prerendered pages, script caches, WebGL shader caches,
   WebRTC bits and pieces, address bar suggestion caches, various networking
   bits that aren’t representations (HSTS/HPKP, SCDH, etc.). Perhaps
-  <a data-link-type="biblio" href="#biblio-storage">[STORAGE]</a> will make this clearer?</p>
+  <span>[STORAGE]</span> will make this clearer?</p>
```

Comment:
Hrm. Do you have an outdated version of Bikeshed locally? Mind updating and trying again?